### PR TITLE
updated the readme file

### PR DESCRIPTION
**Pull request by yahighness**
i changed the readme file and let me know any issues thanks

### CODE REVIEW

File: France/nba_challenge.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+# TODO: Take each player's three point percentage (from part 2.5) and convert it to a string, then print it out.
+# convert to string
+print(str(jamal_murray_3pts_percentage))
+print(str(fred_vanvleet_3pts_percentage))
+print(str(james_harden_3pts_percentage))
+# TODO: Take each player's three point percentage (from part 2.5) and convert it to an integer, then print it out.
+#convert to integer
+print(int(jamal_murray_3pts_percentage))
+print(int(fred_vanvleet_3pts_percentage))
+print(int(james_harden_3pts_percentage))
```
Comment (Sweetpea11) [inline France/nba_challenge.py:96]: I'm so proud of you for completing this challenge. This was not an easy challenge.

File: France/snippets_challenge.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+m = "GOAT"
+n = "goat"
+
+o = (m != n)
+
+print (f"The value of o ({o}) is True since Python is case-sensitive. In order to make statement true, I had to place exclamation mark in front of the equal sign.")
+assert(o == True) #Do not change this line
+
+print()
+print("CHALLENGE COMPLETE!")
```
Comment (Sweetpea11) [inline France/snippets_challenge.py:75]: Yeah, you did it!!!

File: France/restaurants_challenge.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+'''
+Imagine that another one of your most favourite restaurants modified its address. 
+Update just this value in that restaurant's dictionary
+'''
+
+# TODO: Update the address field of 1 restaurant 
+# TODO: Print the new address of the restaurant by accessing that field of the restaurant's dictionary
+# TODO: Print the updated dictionary.
+
+print()
```
Comment (Sweetpea11) [inline France/restaurants_challenge.py:83]: Now that you've gotten better at coding, you can definitely complete this one. Give it a try. If you struggle, review the video and also attend office hours for assistance. Overall, still really proud of your growth.

File: France/do.restaurants_challenge.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+'''
+Imagine that another one of your most favourite restaurants modified its address. 
+Update just this value in that restaurant's dictionary
+'''
+
+# TODO: Update the address field of 1 restaurant 
+# TODO: Print the new address of the restaurant by accessing that field of the restaurant's dictionary
+# TODO: Print the updated dictionary.
+
+print()
```
Comment (Sweetpea11) [inline France/do.restaurants_challenge.py:84]: This is another challenge I'm more that sure you can complete with your skills. Give it a try

File: France/box_office_challenge.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+# TODO: Let's print the top 10 movies in "top_100" as a list. Let's use the slice syntax to accomplish this!
+print(top_100[0:10])
+
+# TODO: Let's convert the list "top_100" back into a string. Assign it to a variable called "top_100_str".
+
+# Note: Print your new variable to see the string version of the movies
+
+top_100_str = str.join(",", top_100)
+
+print(top_100_str)
```
Comment (Sweetpea11) [inline France/box_office_challenge.py:57]: Excellent work!

**Review by Sweetpea11 — APPROVED** [review]

**Comment by yahighness** [comment]
Happy still standing Saturday..going over videos

**Pull request merged**